tools/log-collector/helper.go

```diff
@@ -30,14 +30,17 @@ const (
 	OCPIngress                = "cluster"
 	OCPConfig                 = "default"
 	OCPConfigNs               = "openshift-ingress-operator"
-
-	CoreGv           = "v1"
-	Events           = "events"
-	CRD              = "customresourcedefinitions"
-	Namespaces       = "namespaces"
-	Pod              = "Pod"
-	SubscriptionKind = "Subscription"
-	InstallPlanKind  = "InstallPlan"
+	DefaultNamespace          = "default"
+
+	CoreGv                = "v1"
+	Events                = "events"
+	CRD                   = "customresourcedefinitions"
+	Namespaces            = "namespaces"
+	Pod                   = "Pod"
+	SubscriptionKind      = "Subscription"
+	InstallPlanKind       = "InstallPlan"
+	PersistentVolumeClaim = "persistentvolumeclaims"
+	PersistentVolume      = "PersistentVolume"
```

Comment:
keep naming consistent. check if we need singular or plural name to fetch the objects and use either one but same for both.